plugins/chainlink.prebuilt.Dockerfile

```
@@ -0,0 +1,17 @@
+##
+# Takes Chainlink core as a base image and layers in private plugins.
+##
+ARG BASE_IMAGE=public.ecr.aws/chainlink/chainlink:v2.23.0-plugins
+
+##
+# Final image
+##
+FROM ${BASE_IMAGE} AS final
+# This directory should only contain the plugin binaries.
+ARG LOCAL_PLUGIN_DIR=./plugins
+ARG LOCAL_LIB_DIR=./lib
+
+# Copy/override any additional shared libraries.
+COPY ${LOCAL_LIB_DIR} /usr/lib/
```

Let's make this copy optional? - https://stackoverflow.com/a/46801962

```suggestion
COPY ${LOCAL_LIB_DIR}* /usr/lib/
```

---

Added in 7c6610e863. Made it more strict (for an exact match).